src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -871,24 +871,24 @@ When you are [building a custom decision](#build-decision), you can use our topo
 * Given access to topology feature by a New Relic representative
 * Limit of 10K vertices per New Relic organization per environment
 
-### Topology setup
+### How does topology correlation work? [#topology-explained]
 
-Here's an overview of what needs to be done to set up topology correlation:
+![New Relic topology explained 4](./images/topology-4.png "topology-4.png")
+<figcaption>In this service map, the hosts and apps are the vertices, and the lines showing their relationships are the edges.</figcaption>
 
-1. A topology graph has been created using the NerdGraph aiTopologyCollector APIs.
-2. Vertices with "defining attributes" exist in the topology and they have been connected together with edges to indicate relationship.
-3. The defining attributes on the vertices are also present on your incident events within Incident Intelligence.
+Topology correlation relies on two main concepts:
 
-### How does topology correlation work? [#topology-explained]
+* **Vertices**: Vertices is the plural of vertex. A vertex is a monitored entity and the source from which your incident events are coming from, or describing the problematic symptom about. A vertex has attributes (key/value pairs) configured for it, like entity GUIDs or other IDs, which allow it be associated with incoming incident events.
+* **Edges**: An edge is a connection between two vertices. Edges are used to describe how your vertices are connected together. 
 
-Here's a step-by-step explanation of how topology correlation works:
+To set up your topology, you'll use [our NerdGraph API](#create-topology-graph). But first, it may help to understand how topology correlates incidents: 
```

Comment:
It's not so much that the topology is correlating incidents, but that AI is using topology to correlate incidents more effectively, yeah?